tests/test.js

```diff
@@ -53,3 +53,17 @@ qunit.test('Popup', function (assert) {
 
     assert.equal('<ons-list-item modifier="nodivider"><div class="left list-item__title">Vegan</div> <div class="right list-item__subtitle">foo</div></ons-list-item>', popup.getPopupRows(), 'getPopupRows');
 });
+
+qunit.test('Popup website', function (assert) {
+    'use strict';
+    const popup = new Popup({'diet:vegan': 'foo', 'website': 'https://www.example.com/bar', 'contact:website': 'http://foo.com/'});
+
+    assert.equal('<ons-list-item modifier="nodivider"><div class="left list-item__title">Vegan</div> <div class="right list-item__subtitle">foo</div></ons-list-item><ons-list-item modifier="nodivider"><div class="left list-item__title">Website</div> <div class="right list-item__subtitle"><a target=" blank" rel="noopener" href="https://www.example.com/bar">example.com</a></div></ons-list-item>', popup.getPopupRows(), 'getPopupRows');
```

Comment:
The tests should be updated to take 339a94f14dfc7d17386ff5b498fe191e578b8264 into account (which was recently added on the develop branch):
```
>> Popup website
>> Message: getPopupRows
>> Actual: "<ons-list-item modifier=\"nodivider\"><div class=\"left list-item__title\">Vegan</div> <div class=\"right list-item__subtitle\">foo</div></ons-list-item><ons-list-item modifier=\"nodivider\"><div class=\"left list-item__title\">Website</div> <div class=\"right list-item__subtitle\"><a target=\" blank\" rel=\"noopener\" href=\"https://www.example.com/bar\">example.com</a></div></ons-list-item>"
>> Expected: "<ons-list-item modifier=\"nodivider\"><div class=\"left list-item__title\"><a target=\"_blank\" href=\"https://wiki.openstreetmap.org/wiki/Key%3Adiet:vegan\">Vegan</a></div> <div class=\"right list-item__subtitle\">foo</div></ons-list-item><ons-list-item modifier=\"nodivider\"><div class=\"left list-item__title\">Website</div> <div class=\"right list-item__subtitle\"><a target=\" blank\" rel=\"noopener\" href=\"https://www.example.com/bar\">example.com</a></div></ons-list-item>"
>> at Object.<anonymous> (file:///home/pierre/www/openvegemap/dist/test.bundle.js:23:62759)
```

Reply:
Done